src/dragDrop.js

```diff
@@ -1,38 +1,43 @@
-let current = null;
-let targetItem = null;
+let target;
 
-export function dragStart() {
-  current = this;
-  current.classList.add('current-active');
-}
-
-export function dragEnd() {
-  current.classList.remove('current-active');
-  current = null;
-}
+function sorting(source, target) {
+  const savedList = JSON.parse(localStorage.getItem('savedList'));
```

Comment:
- [ ] I would recommend using ES6 syntax. Use arrow function instead of `function` keyword to make the code cleaner.